package.json

```diff
@@ -53,13 +51,12 @@
     "@types/styled-components": "^4.1.19",
     "@vitejs/plugin-react": "^4.4.1",
     "@vitest/coverage-v8": "^3.1.1",
+    "eslint-config-react-app": "^7.0.1",
     "file-loader": "^6.2.0",
-    "gh-pages": "^5.0.0",
+    "gh-pages": "^6.3.0",
     "jsdom": "^26.1.0",
-    "prettier": "^1.18.2",
-    "tslint": "^5.20.0",
+    "prettier": "^2.8.8",
     "tslint-config-prettier": "^1.18.0",
```

Comment:
This tslint-config-prettier, is outdated, https://github.com/prettier/tslint-config-prettier. No release from 2019.